lib/framebus.js

```diff
@@ -223,8 +223,16 @@ function _broadcast(frame, payload, origin) {
       _broadcast(frame.opener.top, payload, origin);
     }
 
-    for (i = 0; i < frame.frames.length; i++) {
```

Comment:
Seems like you could just change the condition of the `for` instead of re-writing as a `while` loop

```
for (i = 0; frame.frames[i]; i++) {
```